package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "epcis2.js",
-  "version": "0.0.2-beta.1",
+  "version": "0.0.2-beta.2",
```

Comment:
I would rather increment the minor version while leaving the beta flag so: 0.0.3-beta